docs/en/wallet-attestation.rst

```diff
@@ -293,21 +293,27 @@ using the method `POST <https://datatracker.ietf.org/doc/html/rfc6749#section-3.
 The **token** endpoint (as defined in `RFC 7523 section 4`_) requires the following parameters
 encoded in ``application/x-www-form-urlencoded`` format:
 
+* ``OAuth-Client-Attestation`` A JWT that conforms to the structure and syntax defined in the `OAuth 2.0 Attestation-Based Client Authentication` speficification;
+* ``OAuth-Client-Attestation-PoP`` A JWT that conforms to the structure and syntax defined in the `OAuth 2.0 Attestation-Based Client Authentication` speficification;
 * ``grant_type`` set to ``urn:ietf:params:oauth:grant-type:jwt-bearer``;
 * ``assertion`` containing the signed JWT of the Wallet Attestation Request.
 
+* ``OAuth-Client-Attestation and OAuth-Client-Attestation-PoP`` are both defined in `OAuth Attestation Based Client Authentication <https://github.com/oauth-wg/draft-ietf-oauth-attestation-based-client-auth/blob/main/draft-ietf-oauth-attestation-based-client-auth.md>`_.
+
 .. code-block:: http
 
     POST /token HTTP/1.1
     Host: wallet-provider.example.org
     Content-Type: application/x-www-form-urlencoded
+    OAuth-Client-Attestation: ICB7DQogICAgImFsZyI6ICJFUzI1NiIsDQogICAgImtpZCI6ICJ2YmVYSmtzTTQ1eHBodEFObkNpRzZtQ3l1VTRqZkdOem9wR3VLdm9nZzljIiwNCiAgICAidHlwIjogIndhcitqd3QiDQogIH0.ew0KICAiaXNzIjogImh0dHBzOi8vd2FsbGV0LXByb3ZpZGVyLmV4YW1wbGUub3JnL2luc3RhbmNlL3ZiZVhKa3NNNDV4cGh0QU5uQ2lHNm1DeXVVNGpmR056b3BHdUt2b2dnOWMiLA0KICAic3ViIjogImh0dHBzOi8vd2FsbGV0LXByb3ZpZGVyLmV4YW1wbGUub3JnLyIsDQogICJjaGFsbGVuZ2UiOiAiNmVjNjkzMjQtNjBhOC00ZTViLWE2OTctYTc2NmQ4NTc5MGVhIiwNCiAgImhhcmR3YXJlX3NpZ25hdHVyZSI6ICJLb1pJaHZjTkFRY0NvSUF3Z0FJQi4uLnJlZGFjdGVkIiwNCiAgImludGVncml0eV9hc3NlcnRpb24iOiAibzJObWJYUnZZWEJ3YkdVdFlYQndZWC4uLnJlZGFjdGVkIiwNCiAgImhhcmR3YXJlX2tleV90YWciOiAiV1FoeUR5bUZLc1A5NWlGcXB6ZEVEV1c0bDdhVm5hMkZuNEpDZVdIWXRiVT0iLA0KICAiY25mIjogew0KICAgICJqd2siOiB7DQogICAgICAiY3J2IjogIlAtMjU2IiwNCiAgICAgICJrdHkiOiAiRUMiLA0KICAgICAgIngiOiAiNEhOcHRJLXhyMnBqeVJKS0dNbno0V21kblFEX3VKU3E0Ujk1Tmo5OGI0NCIsDQogICAgICAieSI6ICJMSVpuU0IzOXZGSmhZZ1MzazdqWEU0cjMtQ29HRlF3WnRQQklScXBObHJnIiwNCiAgICAgICJraWQiOiAidmJlWEprc000NXhwaHRBTm5DaUc2bUN5dVU0amZHTnpvcEd1S3ZvZ2c5YyINCiAgICB9DQogIH0sDQogICJ2cF9mb3JtYXRzX3N1cHBvcnRlZCI6IHsNCiAgICAgICJqd3RfdmNfanNvbiI6IHsNCiAgICAgICAgImFsZ192YWx1ZXNfc3VwcG9ydGVkIjogWyJFUzI1NksiLCAiRVMzODQiXSwNCiAgICAgIH0sDQogICAgICAiand0X3ZwX2pzb24iOiB7DQogICAgICAgICJhbGdfdmFsdWVzX3N1cHBvcnRlZCI6IFsiRVMyNTZLIiwgIkVkRFNBIl0sDQogICAgICB9LA0KICAgIH0sDQogIH0sDQogICJpYXQiOiAxNjg2NjQ1MTE1LA0KICAiZXhwIjogMTY4NjY1MjMxNQ0KfQ
```

Comment:
the payload contains `\r\n` and anso the kid in the CNF

this JWT should be recreated 


![immagine](https://github.com/user-attachments/assets/94be7995-3ffc-4517-b32d-9aba7227078e)